chroma-manager/tests/integration/utils/chroma_log_collector.py

```diff
@@ -135,46 +135,45 @@ def fetch_log_dir(self, server, dir):
 
         return None
 
-    def fetch_chroma_diagnostics(self, server):
+    def fetch_iml_diagnostics(self, server):
         """
-        Collect the chroma diagnostics from the target
+        Collect the iml diagnostics from the target
         :return: empty list on success or list of error messages that can be used for diagnosing what went wrong.
         """
 
-        # Check that chroma-diagnostics is installed. May not be if installation failed, etc.
-        if shell_run(['ssh', server, 'which chroma-diagnostics']).rc:
-            return["chroma-diagnostics not installed on %s. skipping." % server]
+        # Check that iml-diagnostics is installed. May not be if installation failed, etc.
+        if shell_run(['ssh', server, 'which iml-diagnostics']).rc:
+            return["iml-diagnostics not installed on %s. skipping." % server]
 
         # Generate the diagnostics from the server
-        result = shell_run(['ssh', server, 'chroma-diagnostics', '-v', '-v', '-v'])
+        result = shell_run(['ssh', server, 'iml-diagnostics', '--all-logs'])
```

Comment:
Do we really want `--all-logs` here?

Reply:
It's the closest equivalence to `-v -v -v` at the moment.

Given logs *may* roll over during tests (which I assume happens), I think we do.